src/cloud/paths/tasks.yml

```diff
@@ -176,25 +176,22 @@ post:
 
           - Returns this error if the task doesn't contain one of _`flux`_ or _`scriptID`_.
           - Returns this error if the task contains _`flux`_ _and_ _`scriptID`_.
-      content:
-        application/json:
-          schema:
-            $ref: "../../common/responses/BadRequestError.yml"
-          examples:
-            fluxAndScriptError:
-              summary: The request body can't contain both flux and scriptID
-              value:
-                {
-                  "code": "invalid",
-                  "message": "failed to decode request: can not provide both scriptID and flux"
-                }
-            missingFluxError:
-              summary: The request body requires either flux or scriptID
-              value:
-                {
-                  "code": "invalid",
-                  "message": "failed to decode request: flux required"
-                }
+      $ref: "../../common/responses/BadRequestError.yml"
```

Comment:
@Xenonic Thanks for fixing this! I'm surprised OATS validation let it through. I made a similar mistake yesterday and OATS CI failed it.
FYI, I think `$ref` with siblings isn't technically valid OpenAPI, but it's a rule that we frequently break without problems (AFAIK).